gems/actionpack/6.0/actionpack-generated.rbs

```diff
@@ -511,18 +511,6 @@ module AbstractController
     # sticks the result in <tt>self.response_body</tt>.
     def render: (*untyped args) { () -> untyped } -> untyped
```

Comment:
How about making the block argument optional, too?

Reply:
As for `render`, in my use case I haven't encountered any issues with the current signature. I also haven’t investigated whether the block for `render` is optional. I don't think I'm the right person to make changes here.